app/experimenter/static/rapid/components/experiments/ExperimentEditHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+
+import { useExperimentState } from "experimenter-rapid/contexts/experiment/hooks";
+
+const ExperimentFormPage: React.FC = () => {
+  const { location } = useHistory();
+  let pageHeading = "Create a New A/A Experiment";
+  if (location.pathname.includes("edit")) {
+    const experimentData = useExperimentState();
+    pageHeading = `Edit Experiment: ${experimentData.name}`;
+  }
+
+  return (
+    <div className="mb-4">
```

Comment:
Below this was wrapped in a `<div className="col pt-3">` (see deleted L9 in app/experimenter/static/rapid/components/pages/ExperimentFormPage.tsx).
Not sure if that is still needed, but 🤷 .

Reply:
I intially had it, but it gave me a weird indent 🤷‍♀️. Not exactly the best with front end stuff 😅